application/src/test/java/org/opentripplanner/osm/tagmapping/FinlandMapperTest.java

```diff
@@ -220,4 +225,26 @@ public void testArea() {
     wayData = wps.getDataForWay(way);
     assertEquals(wayData.getPermission(), PEDESTRIAN_AND_BICYCLE);
   }
+
+  @Test
+  public void constantSpeedCarRouting() {
+    OsmTagMapper osmTagMapper = new ConstantSpeedFinlandMapper(20f);
+
+    var slowWay = new OsmWithTags();
+    slowWay.addTag("highway", "residential");
+    assertEquals(20f, osmTagMapper.getCarSpeedForWay(slowWay, true));
+
+    var fastWay = new OsmWithTags();
+    fastWay.addTag("highway", "motorway");
+    fastWay.addTag("maxspeed", "120 kmph");
+    assertEquals(20f, osmTagMapper.getCarSpeedForWay(fastWay, true));
+  }
```

Comment:
I think this should be in a separate test file for the constant speed mapper.

Reply:
I agree, moved.